articles/models/comment.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.contrib.auth.models import User
+from .article import Article
+
+class Comment(models.Model):
+    title = models.CharField(max_length=100, verbose_name='Title')
+    text = models.TextField(max_length=500, verbose_name='Comment body')
+    author = models.ForeignKey(User, on_delete=models.CASCADE, verbose_name='Comment author')
+    article_id = models.ForeignKey(Article, on_delete=models.CASCADE, verbose_name='Article')
```

Comment:
Por consistencia, usaría article en vez de article_id, del mismo modo que una línea más arriba se ha usado author, y no author_id

Reply:
Digamos puse el sufijo id a los campos que son claves foráneas que tienen el mismo nombre que el modelo que buscan. Author como es clave de user no lo puse. La verdad es que es mas mania mía, las pk son id_nombre y las foráneas son nombre_id.